src/ripple/test/mao/impl/Net_test.cpp

```diff
@@ -133,7 +133,7 @@ class Net_test : public beast::unit_test::suite
     }
 };
 
-BEAST_DEFINE_TESTSUITE_MANUAL(Net,mao,ripple)
+BEAST_DEFINE_TESTSUITE(Net,mao,ripple)
```

Comment:
I think we should simply remove the *mao* test, since all the functionality has been moved into `Env`.